### PR TITLE
fix link to tracy

### PR DESCRIPTION
**Pull request by ameknite**
# Objective

 #11331 move the tracy dependency, this remove the line number to avoid keeping the link in sync

### CODE REVIEW

**Review by alice-i-cecile — COMMENTED** [review]
I think we should probably keep pointing to `latest` by default in our docs 🤔

**Comment by ameknite** [comment]
Ok, if we keep it in latest, this should be merge after 0.13

**Comment by hymm** [comment]
We probably should remove the line number rather than trying to keep it in sync.

File: docs/profiling.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The [Tracy profiling tool](https://github.com/wolfpld/tracy) is:
...
 
-It has a command line capture tool that can record the execution of graphical applications, saving it as a profile file. Tracy has a GUI to inspect these profile files. The GUI app also supports live capture, showing you in real time the trace of your app. The version of tracy must be matched to the version of tracing-tracy used in bevy. A compatibility table can be found on [crates.io](https://crates.io/crates/tracing-tracy) and the version used can be found [here](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_log/Cargo.toml#L25).
+It has a command line capture tool that can record the execution of graphical applications, saving it as a profile file. Tracy has a GUI to inspect these profile files. The GUI app also supports live capture, showing you in real time the trace of your app. The version of tracy must be matched to the version of tracing-tracy used in bevy. A compatibility table can be found on [crates.io](https://crates.io/crates/tracing-tracy) and the version used can be found [here](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_log/Cargo.toml#L30).
```
Comment (alice-i-cecile) [inline docs/profiling.md:43]:
```suggestion
It has a command line capture tool that can record the execution of graphical applications, saving it as a profile file. Tracy has a GUI to inspect these profile files. The GUI app also supports live capture, showing you in real time the trace of your app. The version of tracy must be matched to the version of tracing-tracy used in bevy. A compatibility table can be found on [crates.io](https://crates.io/crates/tracing-tracy) and the version used can be found [here](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_log/Cargo.toml).
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Agreed, let's remove the line number and merge this now.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**